src/cli/preprocessors/resize/value.rs

```diff
@@ -59,26 +60,37 @@ impl std::str::FromStr for ResizeValue {
     type Err = anyhow::Error;
 
     fn from_str(s: &str) -> Result<Self, Self::Err> {
+        let s = s.trim().to_lowercase();
+
         match s {
             s if s.starts_with('@') => Ok(Self::Multiplier(s[1..].parse()?)),
             s if s.ends_with('%') => Ok(Self::Percentage(s[..s.len() - 1].parse()?)),
+            s if s.contains('w') && s.contains('h') => Err(anyhow!("Invalid resize value")),
+            s if s.contains('w') => {
```

Comment:
if w is contained, that means user input the height of pic. So, why `(?P<width>\d+)`? it should be `(?P<height>\d+)`

Reply:
Oh no, i misunderstand the role of this arg...